thrift/server.cc

```diff
@@ -302,18 +302,21 @@ thrift_server::requests_served() const {
     return _requests_served;
 }
 
+static const metrics::label class_label("protocol");
+
 thrift_stats::thrift_stats(thrift_server& server) {
     namespace sm = seastar::metrics;
+    auto proto_label_instance = class_label("thrift");
 
-    _metrics.add_group("thrift", {
-        sm::make_derive("thrift-connections", [&server] { return server.total_connections(); },
-                        sm::description("Rate of creation of new Thrift connections.")),
+    _metrics.add_group("transport", {
```

Comment:
I'm tempted to ask to rename this to "coordinator". "transport" doesn't say anythign, it's just the name of the directory where the native protocol lives (which isn't even our native protocol).